libmamba/src/core/package_fetcher.cpp

```diff
@@ -325,23 +325,45 @@ namespace mamba
 
     std::string PackageFetcher::channel() const
     {
-        if (!util::starts_with(m_package_info.package_url, "file://"))
+        // Note that in the oci case, the pkg url also implicitly starts with "https://"
+        // but should be included in the first case
+        // (with the url starting with "https://pkg-containers.githubusercontent.com/" condition)
+        // TODO We should maybe think of a cleaner way of differentiating all this
+        if (!util::starts_with(m_package_info.package_url, "file://")
+            && (!util::starts_with(m_package_info.package_url, "https://")
+                || util::starts_with(
+                    m_package_info.package_url,
+                    "https://pkg-containers.githubusercontent.com/"
+                )))
```

Comment:
Can we come up with a private method for this condition testing OCI (from what I can understand) as well as some background in this PR?